Framework/Crystal/src/SetGoniometer.cpp

```diff
@@ -53,6 +54,38 @@ void SetGoniometer::init() {
   declareProperty("Average", true,
                   "Use the average value of the log, if false a separate "
                   "goniometer will be created for each value in the logs");
+  declareProperty("GoniometerMatrix", "",
```

Comment:
I wonder if this is better defined in a similar way to the UB matrix in `SetUB`
https://github.com/mantidproject/mantid/blob/82b8c6afbdf8d8ca730d83cb2ad40c9c4089977b/Framework/Crystal/src/SetUB.cpp#L70
where
https://github.com/mantidproject/mantid/blob/82b8c6afbdf8d8ca730d83cb2ad40c9c4089977b/Framework/Crystal/src/SetUB.cpp#L42
This would also validate the number of elements supplied.